src/models/supplier-activities/get-device-fields.ts

```diff
@@ -0,0 +1,17 @@
+import Joi from 'joi';
+
+import { schemaConstructor as supplierActivityConstructor, SupplierActivity } from './base';
+
+interface GetDeviceFields extends SupplierActivity<'getDeviceFields'> {
+    triggerData: Record<string, never>;
+}
+
+const schema = supplierActivityConstructor(
+  'getDeviceFields',
+  Joi.object().keys({
```

Comment:
how can one tell what object the deviceFields are for if this is the schema?

Reply:
Removed it as an activity